src/components/FeatureViews/Courses/CourseManagementContainer.js

```diff
@@ -434,20 +436,18 @@ const CourseManagementContainer = () => {
         {defaultCourseDisplay.map(
           ({
             title,
-            dayOfWeek,
+            availabilityList,
             endDate,
-            endTime,
-            startTime,
             startDate,
             instructor,
             id,
           }) => (
             <ClassListItem
               title={title}
-              day={dayOfWeek}
+              day={availabilityList[0].dayOfWeek}
               endDate={endDate}
-              endTime={endTime}
-              startTime={startTime}
+              endTime={availabilityList[0].endTime}
+              startTime={availabilityList[0].startTime}
```

Comment:
If `availabilityList` is an array, it might have multiple time slots so we should handle that accordingly instead of only using the first slot.